src/appleseed-max-impl/appleseedinteractive/interactivesession.h

```diff
@@ -66,15 +67,26 @@ class InteractiveSession
 
     void schedule_camera_update(
         foundation::auto_release_ptr<renderer::Camera>  camera);
+    void schedule_material_update(
+        const IAppleseedMtlMap& material_map);
```

Comment:
Move this arg and the one above on the same line, there's no need to wrap to a new line here.